build_projects/shared-build-targets-utils/Utils/Dirs.cs

```diff
@@ -27,6 +27,7 @@ public static class Dirs
         public static readonly string CorehostLocked = Path.Combine(Output, "corehost", "locked");
         public static readonly string CorehostLocalPackages = Path.Combine(Output, "corehost");
         public static readonly string CorehostDummyPackages = Path.Combine(Output, "corehostdummypackages");
+        public static readonly string SharedFrameworkPublish = Path.Combine(Output, "sharedFrameworkPublish");
```

Comment:
fixed.

---
In reply to: [64611103](https://github.com/dotnet/cli/pull/3230#discussion_r64611103) [](ancestors = 64611103)